File: CursoProgramacionCero/ciclos.js
/* for (let tabla = 1; tabla <= 12; tabla++) {
    for (let num = 1; num <= 12; num++) {
        let resultado = tabla +" * " + num + " = " + tabla*num;
        console.log(resultado);
    }
    console.log("-------------------");
} */


/* for (let par = 0; par <= 12; par++) {
    if (par % 2== 0) {  
        console.log(par + " es par");
    }
    else {
        console.log(par + " es impar");
    }
} */


/* let edad = 0;
while (edad < 18){
    console.log("tienes "+ edad +" años, aun eres un niño");
    edad++
}
console.log("tienes "+ edad +" años, ya eres un adulto"); */


// Blancas = ◻️
// Negras = ⬛

let linea = "";
for (let casilla = 0; casilla < 8; casilla++) {
    for (let fila = 0; fila < 8; fila ++)
        if ((casilla + fila) % 2 === 0) {
            linea += "◻️";
        } else {
            linea += "⬛";
        }
    linea += "\n";  // Salto de línea para la siguiente fila   
}
console.log(linea);

File: CursoProgramacionCero/ciclos.py
""" for tabla in range(1, 13):
    for num in range(1, 13):
        resultado = f"{tabla} * {num} = {tabla*num}"
        print(resultado)
    print("_____________________")  """

""" for par in range(1,13):
    if par %2 == 0:
        print (str(par) + " es par")
    else:
        print (str(par) + " es impar") """

""" edad = 0
while edad < 18:
    print(f"Hola, tienes {edad} años, aun eres menor")
    edad += 1
print(f"Hola, tienes {edad} años, ya eres mayor.")   """  

linea = "";
for casilla in range(1,9):
    for fila in range(1,9):
        if (casilla + fila) % 2 == 0:
            linea += "◻️ "
        else:
            linea += "⬛"
    linea += "\n"

print(linea)